mockgcp/mockapikeys/service.go

```diff
@@ -61,9 +61,11 @@ func (s *MockService) Register(grpcServer *grpc.Server) {
 }
 
 func (s *MockService) NewHTTPMux(ctx context.Context, conn *grpc.ClientConn) (http.Handler, error) {
-	mux := runtime.NewServeMux()
-
-	if err := pb.RegisterApiKeysHandler(ctx, mux, conn); err != nil {
+	mux, err := httpmux.NewServeMux(ctx, conn,
+		pb.RegisterApiKeysHandler,
+		s.operations.RegisterOperationsHandler("v2"),
```

Comment:
I assume this is v2 because https://cloud.google.com/api-keys/docs/reference/rest#rest-resource:-v2.keys is v2. If we plan on this being a template it might be good to document that.

Reply:
Yes, that's the path-prefix that the API implements in front of operation.name.  I'll a comment ...

It will actually become a true template in a few PRs :-)